### DISCUSSION TITLE
Should we remove the Canopy module?

### DISCUSSION THREAD

**Discussion by elenagarrone** [Legal-and-General/canopy#768] [General] [answered]
I'm wondering if we should deprecate the `CanopyModule` as I can't see any scenario where someone would use it. 
And also I think it's easy to add by mistake in a codebase whenever an IDE automatically adds the import.

**Comment by owensgit** [comment, marked as answer]
I would be happy to remove it, it would essentially force precise imports and ensure it was definitely tree-shaken in the build. 

The only time I've ever used it is in quick demo projects so I can just add any component quickly, but that's only a tiny convenience that I wouldn't really miss.

**Comment by pauleustice** [comment]
Agreed - it's better removed
